weather_dl/download_pipeline/partition.py

```diff
@@ -20,7 +20,8 @@
 import apache_beam as beam
 
 from .manifest import Manifest
-from .parsers import Config, prepare_target_name
+from .parsers import prepare_target_name
+from .config import Config
```

Comment:
From looking at how `prepare_target_name` is called, I think it could be a method on the `Config` dataclass. What do you think?

Reply:
It should be kept in `parsers.py` only as moving this in the `Config` dataclass will cause problem of circular import.

Reply:
ah, that makes sense. The status quo is fine, in that case.